src/actions/config.js

```diff
@@ -50,6 +50,9 @@ export function validateConfig(config) {
   if (typeof config.getIn(['backend', 'name']) !== 'string') {
     throw new Error("Error in configuration file: Your `backend.name` must be a string. Check your config.yml file.");
   }
+  if (!isBoolean(config.getIn(['backend', 'squash_merges'], false))) {
```

Comment:
Leave this out, trying to avoid validation of beta features, plus this is specific to one backend. After 2.0 I expect we'll provide a way for specific backends to validate their configuration, as a similar effort is underway for widgets.